libs/python/pythontest/pythontest.go

```diff
@@ -0,0 +1,115 @@
+package pythontest
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"runtime"
+	"strings"
+	"testing"
+
+	"github.com/databricks/cli/internal/testutil"
+	"github.com/databricks/cli/libs/python"
+	"github.com/stretchr/testify/require"
+)
+
+type VenvOpts struct {
+	// input
+	PythonVersion    string
+	skipVersionCheck bool
+
+	// input/output
+	Dir  string
+	Name string
+
+	// output:
+	// Absolute path to venv
+	EnvPath string
+
+	// Absolute path to venv/bin or venv/Scripts, depending on OS
+	BinPath string
+
+	// Absolute path to python binary
+	PythonExe string
+}
+
+func CreatePythonEnv(opts *VenvOpts) error {
+	if opts == nil || opts.PythonVersion == "" {
+		return errors.New("PythonVersion must be provided")
+	}
+	if opts.Name == "" {
+		opts.Name = testutil.RandomName("test-venv-")
+	}
+	if opts.Dir != "" {
+		opts.Dir = "."
+	}
+
+	cmd := exec.Command("uv", "venv", opts.Name, "--python", opts.PythonVersion, "--seed", "-q")
+	cmd.Stdout = os.Stdout
```

Comment:
Isn't it the default for `cmd`?

Reply:
does not seem so: https://pkg.go.dev/os/exec#Cmd

> // If either is nil, Run connects the corresponding file descriptor
> // to the null device (os.DevNull).